### ISSUE TITLE
`@container` should bubble by default

### ISSUE THREAD

**Issue by mayank99** [postcss/postcss-nested#151]
Consider this code:
```css
a {
  color: hotpink;

  @container (min-width: 200px) {
    color: rebeccapurple;
  }
}
```
When running through `postcss-nested`, it seems to remain unchanged.

I would have expected the above to produce:
```css
a {
  color: hotpink;
}
@container (min-width: 200px) {
  a {
    color: rebeccapurple;
  }
}
```

Container queries are now supported in 80% of browsers worldwide, so it makes sense to handle it by default rather than require it to be manually passed to the `bubble` option.

**Comment by ai** [comment]
Fixed 6e3b93c

**Comment by ai** [comment]
Released in 6.1.

**Comment by mayank99** [comment]
awesome, thank you!
